docs/source/conf.py

```diff
@@ -83,7 +83,7 @@
 # Whitelist pattern for branches (set to None to ignore all branches)
 BRANCHES = ['master', 'v1.15', 'v1.16', 'v1.17']
 # Set which versions are not released yet.
-UNSTABLE_VERSIONS = ["master", 'v1.17']
+UNSTABLE_VERSIONS = ["master"]
```

Comment:
we should also change `smv_latest_version` to 1.17 so that `stable` alias points to 1.17

Reply:
Done